api_tests/users/views/test_user_actions.py

```diff
@@ -227,3 +233,7 @@ def test_valid_transitions(self, app, url, preprint, provider, moderator):
                     assert preprint.date_last_transitioned is None
                 else:
                     assert preprint.date_last_transitioned == action.date_created
+
+        # Check if total 8 calls to "get_and_set_preprint_identifiers". The preprint went through total 5 post_moderation in_public_reviews_state
+        # (3 accepted and 2 pending states) and 3 pre_moderation in_public_reviews_state (accepted states).
+        assert mock_ezid.call_count == 8
```

Comment:
Minor: Instead of checking the total (and instead of counting by hand), it'd be nicer to test each iteration whether it was called based on `preprint.in_public_reviews_state`, then do `mock_ezid.reset_mock()`.